plugin/pkg/scheduler/extender.go

```diff
@@ -160,7 +200,7 @@ func (h *HTTPExtender) send(action string, args interface{}, result interface{})
 		return err
 	}
 
-	url := h.extenderURL + "/" + h.apiVersion + "/" + action
+	url := h.extenderURL + "/" + action
```

Comment:
apiVersion is always empty string and so the URL is coming out to be something like http://127.0.0.1:9999/api/scheduler//schedule with double-slash in between where apiVersion was supposed to be. Due to this double-slash, multiple 301 redirects are seen (sent by go router mostly) and extender doesn't even see these scheduling requests, eventually requests fail.